lib/Controllers/create.js

```diff
@@ -30,6 +57,9 @@ Create.prototype.write = function(req, res, context) {
         res.header('Location', location);
       }
 
+      // Reload for that all field are present
+      return instance.reload();
```

Comment:
just remember: this is expensive, you've just made every create in epilogue also do a SELECT afterwards